drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/csv_exporter.rb

```diff
@@ -15,6 +17,25 @@ def initialize(output = StringIO.new)
       self.output = output
     end
 
+    # Input: Hmis::User record
+    # Output: `id` from `User` table, which matches exported User.csv file
```

Comment:
```suggestion
    # Input: `Hmis::User` record (representing an application user in the `users` table)
    # Output: `id` of `Hmis::Hud::User` record from the `User` table, which matches exported User.csv file
```
this stuff always trips me up, more comments may be helpful to future eyes?

Reply:
Also maybe helpful to add some additional comments about why we don't use the existing `from_user` functionality, if you don't think it's too obvious. My understanding is that it's because `from_user` creates a Hud user if there isn't one, whereas in the export we should never be creating users.

Reply:
thanks, updated, added a comment about not wanting to create new users